doc-botonic/docs/concepts/18.webchat.md

```diff
@@ -30,6 +30,156 @@ Botonic supports the following browsers:
+
+```javascript 
+theme: {
+  animations: {
+    enable: false,
+  },
+}
+```
+
+#### Add polyfills
+
+Some functionalities supported by most browsers are not implemented in IE11. To use them, polyfills are needed. 
+
+Here is a list of polyfills used in some of the bots:
+
+- Basic polyfills: [core-js](https://www.npmjs.com/package/core-js)
+- Style issues: [@webcomponents/webcomponentsjs](https://www.npmjs.com/package/@webcomponents/webcomponentsjs)
+- scrollTo feature: [element-scroll-polyfill](https://www.npmjs.com/package/element-scroll-polyfill)
+- normalize: [unorm](https://www.npmjs.com/package/unorm)
+
+To add polyfills:
+
+1. Open the `webpack.config.js` file.
+
+2. Add all the polyfills that are currently used in the entry property within the `botonicWebchatConfig`. The first 2 polyfills are mandatory, as well as the 2 that follows:
+
+```javascript
+entry: [
+  'core-js/stable/',
+  'regenerator-runtime/runtime',
+  '@webcomponents/webcomponentsjs',
+  'element-scroll-polyfill',
+  'unorm',
+  path.resolve('src', 'webchat-entry.ts'),
+],
+```
+
+#### Configure the Babel loader
+
+When configuring the Babel loader to transpile code to ES5 compliance, some options must be added/configured.
+
+1. Set `sourceType` to 'unambiguous' (cf. issue: https://github.com/babel/babel/issues/8900 )
+
+2. In presets, the definition order is important. Therefore, `babel/env` preset needs to be set before typescript to avoid losing constructor auto-assign: (https://github.com/babel/babel/issues/8752#issuecomment-486541662)
+
+3. Add plugins to transform some new functionalities from ES6 to ES5 compliant. For example: [the arrow functions](https://www.npmjs.com/package/@babel/plugin-transform-arrow-functions)
+
+```javascript
+options: {
+  sourceType: 'unambiguous',
+  cacheDirectory: true,
+  presets: [
+    '@babel/react',
+    [
+      '@babel/env',
+      {
+        targets: {
+          ie: '11',
+        },
+        useBuiltIns: 'entry',
+        corejs: 3,
+        modules: false, // Needed for tree shaking to work.
+      },
+    ],
+    '@babel/typescript',
+  ],
+  plugins: [
+    require('@babel/plugin-proposal-object-rest-spread'),
+    require('@babel/plugin-proposal-class-properties'),
+    //require('babel-plugin-add-module-exports'),
+    require('@babel/plugin-transform-runtime'),
+    require('@babel/plugin-transform-object-assign'),
+    require('@babel/plugin-transform-arrow-functions'),
+    require('@babel/plugin-transform-classes'),
+  ],
+}
+```
+
+#### Load the bot and correct the dependencies
+
+You can now load the bot in IE11. If an error still occurs despite the correct configuration in `webpack.config.js`, you can solve them by finding the error and the dependency that produces the error.
+
+To do so:
+
+1. Open the developer tool (F12).
+2. Click on the error messages line to display the exact part of the code where the error occurs.
+3. Try to find some parts of the code that are easy to search.
```

Comment:
what does "easy to search" mean?

Reply:
What I wanted to say here is to find parts of code that are unique/exclusive of a package (function names, class names, text messages, etc.) so you can search that part in `node_modules` and know which dependency is failing.

Reply:
st like this would make it eaier to understand, thanks